### ISSUE TITLE
Stream with mpg & how handle the output

### ISSUE THREAD

**Issue by ChristianOrgs** [dominictarr/mpg123#7]
Is there an option to handle the output of mpg? 
I need the Icy-Cast informations like the song name.
If I start it with the terminal I will get this as well but cant figure out where this informations are holded.

With "player.track"? I just get "stream".

Thanks in advance.

**Comment by dominictarr** [comment]
No, it's just a very basic wrapper. make a PR and I'll add you so you can merge it
